f3/granite.go

```diff
@@ -677,6 +662,11 @@ func (i *instance) beginDecide(round uint64) {
 		panic("beginDecide with no strong quorum for value")
 	}
 
+	// DECIDE messages always specify round = 0.
```

Comment:
If they specify round=0, why do we have the special case for `math.MaxUint64`?

Reply:
It's because the DECIDE round does not match the justification COMMIT round, nor can be calculated from it. So in the expectations it's a sentinel to say "don't check the justifying COMMIT round".